src/contents/en-gb/doc_xamarin.md

```diff
@@ -308,11 +308,11 @@ var encodedAuthReq = await GenerateEncodedAuthReqAsync();
 var url = UrlFormat.Format(Constants.AppId, encodedAuthReq.Item2, true);
 Device.BeginInvokeOnMainThread(() => { Device.OpenUri(new Uri(url)); });
 ```
-2. **Grant access**
+**2. Grant access**
```

Comment:
@nadiaburborough Can you please add a new line with `<br />` before this title. This will add one line space between code block and the heading.